flow/flow.troll.logfiles.fill/wrap.troll.logfiles.fill.R

```diff
@@ -232,19 +247,19 @@ wrap.troll.logfiles.fill <- function(DirInLogs=NULL,
     if(sensor=='leveltroll500'){
       keep_flags<-c('readout_time','pressureLogFlag','logDateFlag','temperatureLogFlag')
       flagsOut<-dataOut[keep_flags]
-      keep_data<-c('source_id.x','site_id','readout_time','pressure.x','pressure_data_quality','temperature.x','temperature_data_quality','internal_battery')
+      keep_data<-c('source_id.x','site_id.x','readout_time','pressure.x','pressure_data_quality.x','temperature.x','temperature_data_quality.x','internal_battery.x')
       dataOut<-dataOut[keep_data]
-      names(dataOut)<- c('readout_time','source_id','site_id','pressure','pressure_data_quality','temperature','temperature_data_quality','internal_battery')
+      names(dataOut)<- c('readout_time','source_id','site_id.x','pressure','pressure_data_quality','temperature','temperature_data_quality','internal_battery')
```

Comment:
Looks like this is an intentional change, but just a heads up that site_id.x likely doesn't match the schema. No need to change if that's how you want it.